README.md

```diff
@@ -1,37 +1,70 @@
 # rbflagr
```

Comment:
Should the use of `Makefile` be documented here?

Reply:
This file will be auto generated every time; I can put the doc in Makefile and append them here

Reply:
I didn't realize the `README` was autogenerated, I don't have strong feelings about the need to document the Makefile given that fact (the Makefile is simple enough anyway).